File: CodeFreak1/ClientApp/src/app/Security/Dtos/user-roles-view-model.ts
import { RequestStatus } from "./request-status";
import { UsersViewModel } from "./users-view-model";
import { RolesPermissionsViewModel } from "./roles-permissions-view-model";

export class UserRolesViewModel extends RequestStatus {
  public User: UsersViewModel;
  public RolePermissions: Array<RolesPermissionsViewModel>;
  public Token: string;
}

File: CodeFreak1/ViewModel/UserRolesViewModel.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace CodeFreak1.ViewModel
{
    public class UserRolesViewModel : RequestStatus
    {
        public UsersViewModel User { get; set; }
        public List<RolesPermissionsViewModel> RolePermissions { get; set; }
        public string Token { get; set; }

    }
}
